### ISSUE TITLE
great idea!

### ISSUE THREAD

**Issue by Rik44444** [krestaino/andotp-extension#1]
just wanted to say i really like this idea and support it here also. 
for me as a user, priorities for the future would be:
- Firefox version (you write Edge too, but i believe it is Chromium based these days, so should work with the Chrome version?)
- encryption
- sync 

can i already start using the extension as a regular user?

**Comment by cdorin93** [comment]
This looks very promising. Any chance to hear about the latest developments? ^^ . Would also be more than happy to buy you a coffee :+1:

**Comment by Rik44444** [comment]
yep i'm happy to contribute too. Right now i use Authy, it's very good, but i would prefer an open-source alternative!
